prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -390,6 +414,34 @@ class FHIRRouter(
         return if (reverseFilter) !result else result
     }
 
+    /**
+     * Log the results of filtering items out of reports during the "route" step
+     *
+     * @param filters - Filters run on the items
```

Comment:
Change this kdoc to have the parameters online.  This is what the program wanted.

Reply:
Done